### PR TITLE
🎨✨About page hero section #183

### LINKED ISSUE #183: 🎨 Create About Page Hero Section
#  Create the about page hero section 
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a nice hero section on the about page that captures my attention and introduces the page content. Reuse the text-content component created in #20 and the image container component created in #19. I want a good experience on the web and phone.

Design:

about home

Link to full design:

```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```
 
### Technical Analysis

This feature can be found in `libs/features/pages/about` inside a components folder. The component's name is about-hero-section.

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 30 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the global style.scss
5. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by PerisOduol618**
# Description

A hero section on the about page that introduces the page content

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/elewa-website/issues/183).

Fixes # (issue) - *Example*: Fixes #183 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

- [x ] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
Screenshot from 2023-09-07 16-06-20

## Checklist

- [ X] When rendered the component is pixel perfect compared to the design above.
- [X ] The component is clean and does not have more than 30 lines
- [X ] The component SCSS uses CSS variables for colour, font, ... Connected to the global style.scss
- [X ]The design is responsive for screen, tablet and phone

### CODE REVIEW

File: libs/features/pages/about/src/lib/components/about-hero-section/about-hero-section.component.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+}
+
+.about-hero-text {
+    flex: 1;
+    letter-spacing: 0px;
+    align-items: flex-start;
+    padding-left: 40px;
+    text-align: left;
+    color: #000000;
```
Comment (AnnaL001) [inline libs/features/pages/about/src/lib/components/about-hero-section/about-hero-section.component.scss:19]: For reuse of colors, use CSS variables specified in elements/theming/_variables.scss for colors/background colors

File: libs/features/pages/about/src/lib/components/about-hero-section/about-hero-section.component.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  styleUrls: ['./about-hero-section.component.scss'],
+})
+export class AboutHeroSectionComponent {
+  heroAboutData = {
+    title: 'Lorem ipsum dolorem sit eos',
+    subtitle: '',
+    description: 'Neque porro quisquam dolorem ipsum quia dolor sit Eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam dolorem ipsum quia dolor sit Eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam dolorem ipsum quia dolor.'
+  };
+  image= __elewaImageData
+
```
Comment (AnnaL001) [inline libs/features/pages/about/src/lib/components/about-hero-section/about-hero-section.component.ts:17]: Just like the 'image' variable, the 'heroAboutData' variable content can be moved to the same Typescript file as the 'heroAboutData' and referenced from there.

File: libs/features/pages/about/src/lib/features-pages-about.module.ts [hunk 3]
```diff
@@ -8,10 +8,14 @@ import { CarouselModule } from '@elewa-website/elements/layout/carousel';
...
 
 import { AboutPageComponent } from './main/about-page/about-page.component';
-import { TeamMembersSectionComponent } from './components/team-members-section/team-members-section.component';
 import { DiscoverSectionComponent } from './components/discover-section/discover-section.component';
 
 import { AboutRoutingModule } from './about.routing';
+import { TeamMembersSectionComponent } from './components/team-members-section/team-members-section.component';
+import { AboutHeroSectionComponent } from './components/about-hero-section/about-hero-section.component';
+import { ImagesModule } from '@elewa-website/elements/layout/images';
+import { TextsModule } from '@elewa-website/elements/layout/texts';
```
Comment (AnnaL001) [inline libs/features/pages/about/src/lib/features-pages-about.module.ts:17]: Kindly, observe the order of imports; elements imports should appear before local imports

**Review by AnnaL001 — CHANGES_REQUESTED** [review]
Good work, just a few changes

**Comment by PerisOduol618** [comment]
@AnnaL001 Done

**Review by LemmyMwaura — APPROVED** [review]

**Pull request merged**